components/Layout/logoHeader.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import scss from '../../pageStyles/layout.module.scss';
+
+const LogoHeader = () => (
+    <img className={scss.logoHeader} alt="TechSwitch" src="https://www.techswitch.co.uk/images/logo.svg">
```

Comment:
I think it might be better to copy this image into our own source code.
Otherwise if the techswitch site changes how it loads images etc then our site would break.